client-v2/src/actions/Clipboard.ts

```diff
@@ -56,11 +57,23 @@ function storeClipboardContent(clipboardContent: NestedArticleFragment[]) {
 
 function updateClipboard(clipboardContent: {
   articles: NestedArticleFragment[];
-}): ThunkResult<Promise<NestedArticleFragment[] | void>> {
-  return () =>
-    saveClipboard(clipboardContent.articles).catch(() => {
+}): ThunkResult<Promise<NestedArticleFragment[]>> {
+  return async (_, getState: () => State) => {
+    const saveClipboardResponse = await saveClipboardStrategy(
+      getState(),
+      clipboardContent.articles
+    );
+    try {
+      if (!saveClipboardResponse) {
```

Comment:
Why the try/catch here -- it doesn't look like it's possible for this code to throw?